### ISSUE TITLE
"pilot change" breaks if there are other rewrite actions available

### ISSUE THREAD

**Issue by pokey** [C-Loftus/talon-ai-tools#12]
See https://github.com/orgs/community/discussions/103149. I believe this is out of our control for now, but once that discussion is addressed we should be able to just switch to the more specific action kind here

**Comment by wolfmanstout** [comment]
I've been using:

```
actions.user.vscode("inlineChat.start")
```

instead of:
```
actions.user.run_rpc_command(
            "editor.action.codeAction",
            {
                "kind": "refactor.rewrite",
            },
        )
```

It seems to work fine. Shall I send a PR or is there a concern with this approach?

**Comment by C-Loftus** [comment]
Nope, feel free to submit a PR, that works for me! If I recall I think that rpc command syntax may have not existed at the time the pilot commands were written. But can't recall for sure
